src/autometrics/tracker/opentelemetry.py

```diff
@@ -146,5 +150,9 @@ def finish(
         objective: Optional[Objective] = None,
     ):
         """Finish tracking metrics for a function call."""
-        self.__count(function, module, caller, objective, result)
-        self.__histogram(function, module, start_time, objective)
+        exemplar = None
+        # Currently, exemplars are only supported by prometheus-client
```

Comment:
I think it might be good to keep an issue open for this. So we can refer to it from our source code and also have a place to talk about how to deal/resolve this

Reply:
we actually have one! i added a backlink there and also linked the issue from code